src/tools/interop/idt/capture/platform/README.md

```diff
@@ -0,0 +1,16 @@
+# Platform
+
+This directory contains the logging transport implementation for various
+platforms.
+
+## Adding new platforms
+
+1. Add the platform package in this directory (`/platform`).
```

Comment:
Is someone able to add in a new extension or plugin without modifying the tool? It feels like the entry-point needs to be able to accept some information about additional plugins. There will probably be users that want to write their own plugins but are unable to share it with the public.

Reply:
When I originally wrote this, I did a more dynamic search of the contents of `/ecosystem` (instead of the simple reliance on `__all__` of the package that is in `factory` today).

I could put this back, and users could symlink an external file into `/ecosystem` as the only step needed to extend the tool with an external dir. Does that sound sufficient?

Reply:
The comment was a thought exercise aoubt how to structure the tool framework so that external extensions could be added without directly modifying the core.

I didn't know, but, I just asked ChatGPT and the answer seems to make sense to me.

[What is a good way to structure a python library to allow external extensions or plugins to be registered during initialization and used during runtime?](https://chat.openai.com/share/d6466bec-0fc4-41bd-b4d3-8435609be842)